CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+## Release 1.1.0
+### Summary
+This is a small release in order to add facts, functions, tasks and plans.
+
+### Added
+- Functions added for onverting to and from bytes.
+- Tasks added for common actions that use types and providors.
+- Plan for one touch expanding, expanded.
+- Support has been added for LVM Thin Volumes.
+
+### Changed
```

Comment:
Changed header means 'Backwards Incompatible' therefore these are probably features.

Reply:
'Thin Volumes' I dont think you need the uppercase here.

Reply:
Changed to `LVM thin volumes`

Reply:
`Added` and `Changed` merged into `Features`. `Fixed` changed to `Bugfixes`